portfolio/src/main/webapp/api-script/script.js

```diff
@@ -0,0 +1,87 @@
+mapOptions = {
+  center: { lat: 0, lng: 0 },
+  zoom: 3,
+  scaleControl: false,
+  streetViewControl: false,
+  mapTypeControlOptions: {
+    mapTypeIds: ["moon"]
+  }
+};
+
+
+async function loadMap() {  
+  const response = await fetch("/landing-data");
+  const payload = await response.json();
+
+  
+  var map = new google.maps.Map(document.getElementById("map"), mapOptions);
+
+  var moonMapType = new google.maps.ImageMapType({
+    getTileUrl: function (coord, zoom) {
+      var normalizedCoord = getNormalizedCoord(coord, zoom);
+      if (!normalizedCoord) {
+        return null;
+      }
+      var bound = Math.pow(2, zoom);
+      return (
+        "https://mw1.google.com/mw-planetary/lunar/lunarmaps_v1/apollo" +
+        "/" +
+        zoom +
+        "/" +
+        normalizedCoord.x +
+        "/" +
+        (bound - normalizedCoord.y - 1) +
+        ".jpg"
+      );
+    },
+    tileSize: new google.maps.Size(256, 256),
+    maxZoom: 19,
+    minZoom: 0,
+    radius: 1738000,
+    name: "Moon"
+  });
+
+  map.mapTypes.set("moon", moonMapType);
+  map.setMapTypeId("moon");
+
+  var infowindow = new google.maps.InfoWindow();
+
+  var marker;
+
+  for (i in payload) {
+    const info = payload[i];
+    console.log(info);
+    const imgRef = "http://mw1.google.com/mw-planetary/lunar/lunarmaps_v1/util/s".concat(info.missionNum).concat(".png");
```

Comment:
Would also use string interpolation here: https://dmitripavlutin.com/string-interpolation-in-javascript/